src/connect/components/tree-view/tree-view.tsx

```diff
@@ -54,26 +50,20 @@ export function ConnectTreeView(props: ConnectTreeViewProps) {
     return (
         <>
             {items.map(item => {
-                if (
+                const mode =
                     item.action === ActionType.New ||
                     item.action === ActionType.Update
```

Comment:
you should also consider `UpdateAndMove` and `UpdateAndCopy` here.

https://github.com/powerhouse-inc/design-system/blob/main/src/connect/components/tree-view-item/tree-view-item.tsx#L25-L30